### PR TITLE
[DUOS-2026][risk=no] various light bug fixes

### PR DESCRIPTION
**Pull request by lu-c**
-[fixed unmounted support request modal object memory leak](https://github.com/DataBiosphere/duos-ui/commit/3fb57a74ef22c580db2cf3324884d9458a9b7c7e)
-[[DUOS-2026] fixed "uncaught exception" error](https://github.com/DataBiosphere/duos-ui/commit/f26b5d705b456556a97a02b4dc0e4a18135e5f09)
- [[noticket] Fixed bug with:](https://github.com/DataBiosphere/duos-ui/commit/b1b323f498ea40c5dd6dc9c21cd35dbf85d3402b) 

Warning: Each child in a list should have a unique "key" prop.

[DUOS-2026]: https://broadworkbench.atlassian.net/browse/DUOS-2026?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/DuosHeader.js [hunk 1]
```diff
@@ -386,7 +386,7 @@ class DuosHeader extends Component {
       'data-tip': 'Need help? Contact us for some assistance', 'data-for': 'tip_requestAccess'
     }, [contactUsIcon, contactUsText]);
 
-    const supportrequestModal = SupportRequestModal({
+    const supportrequestModal = h(SupportRequestModal, {
```
Comment (lu-c) [inline src/components/DuosHeader.js:389]: this line fixes the "unmounted component exception" that occasionally shows up

File: src/components/institution_table/InstitutionTable.js [hunk 2]
```diff
@@ -72,7 +72,7 @@ export default function InstitutionTable(props) {
   return (
     div({className: 'institution-table-component'}, [
       div({style: Styles.TABLE.CONTAINER}, [
-        div({style: Styles.TABLE.HEADER_ROW}, [tableHeaderTemplate]),
+        div({style: Styles.TABLE.HEADER_ROW}, tableHeaderTemplate),
```
Comment (lu-c) [inline src/components/institution_table/InstitutionTable.js:75]: this line fixes the "unique key in table" error that shows up when you click on the admin institution tab

File: src/libs/notificationService.js [hunk 3]
```diff
@@ -30,8 +30,12 @@ export const NotificationService = {
    * @returns {Promise<JSON>}
    */
   getActiveBanners: async () => {
-    const banners = await NotificationService.getBanners();
-    return filter(banners, {active: true});
+    try {
```
Comment (lu-c) [inline src/libs/notificationService.js:33]: this block fixes the "uncaught exception" error that shows up when you reload the page

**Review by JVThomas — APPROVED** [review]
Looks good to me!

File: src/components/DuosHeader.js (hunk 1 above)
Comment (connorlbark) [reply src/components/DuosHeader.js:389]: hmm: I am still getting this error ??
image

Comment (lu-c) [reply src/components/DuosHeader.js:389]: hopefully less at least... maybe this same issue is in multiple places :X

**Review by connorlbark — APPROVED** [review]
we can investigate the unmounted component isssue more later - but looks good otherwise!

**Pull request merged**